docs/UserGuide.md

```diff
@@ -950,6 +1022,13 @@ been cancelled or completed. If that's the case, simply use this feature to dele
 
 **Format:** `delivery delete DELIVERY_ID`
 
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+
+* `DELIVERY_ID`{.swift} must be an integer greater than 0.
```

Comment:
`DELIVERY_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Delivery in the address book.